### PR TITLE
Allow chained CSS classes

### PR DESCRIPTION
**Pull request by readingdancer**
This fixes issue: #3 

In our SCSS Icon font file we have some icons defined like this:

```
.swicon-linkedin
{ 
    &:before {
    content: "\f0e1";
    }
    &.color:before {
        color: #0077b5 !important;
    }
}
```

This broke the display of the icons in the back office.

This pull requests allows the .color class to work correctly in the back office UI by removing the dots when adding to the class statement, this would also support multiple CSS classes, eg. the icon, color & size.

You can see in the screen shot how this is working:

screenshot 39

To summarize:

Before:

class="icon.color"

After:

class="icon color"

### CODE REVIEW

**Comment by bgunnarsson** [comment]
Thanks for your help @readingdancer I'll be updating the NuGetpackage as well.
